src/components/common/TmSessionWelcome.vue

```diff
@@ -79,7 +79,14 @@ export default {
   },
   methods: {
     setState(value) {
-      this.$store.commit(`setSessionModalView`, value)
+      if (navigator.userAgent.includes(`Chrome`)) {
+        this.$store.commit(`setSessionModalView`, value)
+      } else {
+        this.$store.commit(`notifyError`, {
```

Comment:
notifyError is a better we avoid these days. Can you better show a message inline?
also the problem is only signing in with Ledger, here you throw an error whenever the user clicks any one of the sign in options. I would recommend to show the warning only on the sign in screen with Ledger. And when the user tryes to sign a transaction in the ActionModal.

Reply:
@faboweb Thanks for the feedback. 

I've fixed this so that the error message only occurs when an user has selected the Ledger option. I've also applied this to the ActionModal component.